### DISCUSSION TITLE
Quote Tax Mode and Discount Mode need to select as default Individual

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14724] [How to] [answered]
Sir, I need to set Tax Mode and Discount Mode to default Individual for user. I read issue https://github.com/YetiForceCompany/YetiForceCRM/issues/1966 but not able to configure

![image](https://user-images.githubusercontent.com/12723595/119532438-804fd800-bda2-11eb-9ff0-efcb79e0e270.png)

Kindly assist.

Regards
Jeevan Pendhari

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
there is no such possibility, but you have a little gift.
I added it, look at it
https://github.com/YetiForceCompany/YetiForceCRM/commit/747b122a5cb0d7fbaac537ee61a9fabc03e74f11

config\Modules\SQuotes.php
```php
<?php

/**
 * Configuration file.
 * This file is auto-generated.
 *
 * @package Config
 *
 * @copyright YetiForce Sp. z o.o
 * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
 */

namespace Config\Modules;

/**
 * Configuration file: Config\Modules\SQuotes.
 */
class SQuotes
{
	/** List of fields read from related module */
	public static $INVENTORY_ON_SELECT_AUTO_COMPLETE = [
		'description' => ['ref' => 'getInventoryListName'],
		'price' => ['ref' => 'getInventoryPrice'],
	];

	public static $defaultInventoryData = [
		'discountmode' => 1,
		'taxmode' => 1,
	];
}
```

**Comment by JEEVAN28** [comment]
@mariuszkrzaczkowski Bingo, your gift is sweat. Thanks for the help.

Regards
Jeevan Pendhari
